espresso-support/demo/src/debug/res/layout/test_movie_item_view.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.novoda.movies.MovieItemView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
we wanted to do it so that you didn't even have to specify a layout file. Most of the custom views at Novoda have logic in `onFinishInflate()` (e.g. dependency/collaborator instantiation) and `onFinishInflate()` will not be called unless the view was inflated from XML.